app/graphql/types/user_proxy_type.rb

```diff
@@ -3,12 +3,13 @@ class UserProxyType < Types::BaseObject
     field :id,
           ID,
           null: false,
-          description:
-            'The ID returned by the type can represent different user role types'
+          description: 'The ID returned by the type can represent different user role types'
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)